### DISCUSSION TITLE
Tid is na after using logback plugin

### DISCUSSION THREAD

**Discussion by ghost** [apache/skywalking#7631] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

![image](https://user-images.githubusercontent.com/8934662/131624101-837606c7-fbba-4bec-b64c-383dd2f81371.png)

### Use case

I don't know where the configuration is wrong ，why tid is na

### Related issues

I don't know where the configuration is wrong ，why tid is na

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
You didn't submit report correctly, this is not a feature. This should be only a discussion.

**Reply by ghost** [reply]
Can you help me see why this tid is na, thank you

**Comment by wu-sheng** [comment]
I am not sure what do you ask for. We never said there always is trace id in logs. Only logs in instrumented thread context have trace id.
